ipn/ipnlocal/local.go

```diff
@@ -1216,20 +1218,11 @@ func (b *LocalBackend) SetControlClientStatus(c controlclient.Client, st control
 
 	if st.NetMap != nil {
 		if envknob.NoLogsNoSupport() && st.NetMap.HasCap(tailcfg.CapabilityDataPlaneAuditLogs) {
-			msg := "tailnet requires logging to be enabled. Remove --no-logs-no-support from tailscaled command line."
+			msg := "tailnet requires logging to be enabled. Remove --no-logs-no-support from the tailscaled command line."
```

Comment:
Does this message get surfaced in the admin panel UI?

Reply:
Well, in theory. That's what the following line should be doing indirectly. It sets health as unhappy on the device and nodes should be telling control about their health state. But we don't do much with that server-side yet.